lib/Dependency.js

```diff
@@ -43,10 +43,6 @@ class Dependency {
 	disconnect() {
 		this.module = null;
 	}
-
-	compare(a, b) {
-		return compareLocations(a.loc, b.loc);
-	}
```

Comment:
This is here for compatibility. Instead of deleting, add a comment `TODO remove in webpack 3`